<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1221</width>
    <height>42</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QPushButton" name="btnHome">
     <property name="text">
      <string>Home</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnProfile">
     <property name="text">
      <string>Profile</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnSettings">
     <property name="text">
      <string>Settings</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnStats">
     <property name="text">
      <string>Stats</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
